tests/suites/test_suite_ssl.data

```diff
@@ -199,6 +199,42 @@ move_handshake_to_state:MBEDTLS_SSL_IS_CLIENT:MBEDTLS_SSL_SERVER_HELLO_VERIFY_RE
 Negative test moving servers ssl to state: NEW_SESSION_TICKET
 move_handshake_to_state:MBEDTLS_SSL_IS_SERVER:MBEDTLS_SSL_SERVER_NEW_SESSION_TICKET:0
 
+Handshake, SSL3
+depends_on:MBEDTLS_SSL_PROTO_SSL3:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED
+handshake:"":MBEDTLS_SSL_MINOR_VERSION_0:MBEDTLS_PK_RSA
```

Comment:
Does this really depend on MBEDTLS_ECP_DP_SECP384R1_ENABLED?

Reply:
Without this define every test fails, because in `mbedtls_endpoint_certificate_init` three `mbedtls_test_cas` are parsed, and the third one is `mbedtls_test_ca_crt_ec`.

Reply:
But this test case is only using RSA, not ECP, right?

Reply:
This issue has been described here:
https://github.com/ARMmbed/mbedtls/issues/3032
I'll probably raise a PR later on to fix this.